costs/tags/tags_values.go

```diff
@@ -88,12 +89,12 @@ func getTagsValuesWithParsedParams(ctx context.Context, params tagsValuesQueryPa
 		if returnCode == http.StatusOK {
 			return returnCode, response
 		}
-		return returnCode, errors.New("Internal server error")
+		return returnCode, err
```

Comment:
Same as above.

Reply:
`GetErrorMessage` is already called in `makeElasticSearchRequestForTagsKeys`